common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -59,10 +59,10 @@ trait FeatureSwitches {
 
   val PlistaForAU = Switch(
     SwitchGroup.Feature,
-    "plista-for-outbrain-au",
-    "Enable the Plista content recommendation widget to replace that of Outbrain for AU edition (for web only).",
-    owners = Seq(Owner.withGithub("JonNorman")),
-    safeState = Off,
+    "plista-for-au",
+    "Enable the Plista content recommendation widget in Australia",
+    owners = group(Commercial),
+    safeState = On,
```

Comment:
It seems safer to me to have the `safeState` as Off. (pun intended)

Reply:
Agree, in this case I turned it to On because with changing its name the switch also looses its previous state (which was On in PROD) and wanted to avoid having plista not running even for the little time between deploy and manually switching in On. 
I can create another PR after deploy to change it to Off

Reply:
That sounds reasonable to me. 👍 It should prevent Plista from not running immediately after this build goes live.